modules/identity/src/Volo.Abp.Identity.Domain/Volo/Abp/Identity/IdentityRoleManager.cs

```diff
@@ -43,5 +44,35 @@ public async Task<IdentityRole> GetByIdAsync(Guid id)
 
             return role;
         }
+
+        public List<IdentityRole> GetDefaultRoles()
+        {
+            if (Roles == null)
+            {
+                return new List<IdentityRole>();
+            }
+
+            return Roles.Where(r => r.IsDefault).ToList();
+        }
+
+        public override async Task<IdentityResult> SetRoleNameAsync(IdentityRole role, string name)
+        {
+            if (role.IsStatic && role.Name != name)
+            {
+                throw new AbpException("Static roles can not be renamed."); // TODO: localize & change exception type
```

Comment:
Convert to business exception and localize. Documentation: https://abp.io/documents/abp/latest/Exception-Handling#business-exceptions